code/modules/relay/relay.dm

```diff
@@ -0,0 +1,53 @@
+/datum/config_entry/string/ooc_webhook
+	protection = CONFIG_ENTRY_LOCKED | CONFIG_ENTRY_HIDDEN
+
+/datum/config_entry/string/adminhelp_webhook
+	protection = CONFIG_ENTRY_LOCKED | CONFIG_ENTRY_HIDDEN
+
+/datum/config_entry/str_list/webhook_allowed_mention_types
+	protection = CONFIG_ENTRY_LOCKED
+
+/datum/config_entry/str_list/webhook_allowed_mention_users
+	protection = CONFIG_ENTRY_LOCKED
+
+/datum/config_entry/str_list/webhook_allowed_mention_roles
+	protection = CONFIG_ENTRY_LOCKED
+
+/// Send OOC via webhook. allowed_x arguments are lists of strings, bypassing restrictions on mentions, joined to the values in the config.
+/proc/sendooc2ext(msg, allowed_types = list(), allowed_users = list(), allowed_roles = list())
+	var/ooc_webhook = CONFIG_GET(string/ooc_webhook)
+	if(!length(ooc_webhook))
+		return
+	send_webhook(ooc_webhook, msg)
+
+/// Send adminhelp via webhook. allowed_x arguments are lists of strings, bypassing restrictions on mentions, joined to the values in the config.
+/proc/sendadminhelp2ext(msg, allowed_types = list(), allowed_users = list(), allowed_roles = list())
+	var/adminhelp_webhook = CONFIG_GET(string/adminhelp_webhook)
+	if(!length(adminhelp_webhook))
+		return
+	send_webhook(adminhelp_webhook, msg)
+
+/// Send text via webhook, asynchronously. allowed_x arguments are lists of strings, bypassing restrictions on mentions, joined to the values in the config.
+/// sent as JSON via POST {"content": "[msg]", "allowed_mentions": {"parse": [...]}}
+/proc/send_webhook(link, msg, allowed_types = list(), allowed_users = list(), allowed_roles = list())
```

Comment:
```suggestion
/proc/send_webhook(link, msg, list/allowed_types = list(), list/allowed_users = list(), list/allowed_roles = list())
```
Note that this function should NOT be passed in null, because `|` can't handle null